tx_check.go

```diff
@@ -0,0 +1,201 @@
+package bbolt
+
+import (
+	"encoding/hex"
+	"fmt"
+)
+
+// Check performs several consistency checks on the database for this transaction.
+// An error is returned if any inconsistency is found.
+//
+// It can be safely run concurrently on a writable transaction. However, this
+// incurs a high cost for large databases and databases with a lot of subbuckets
+// because of caching. This overhead can be removed if running on a read-only
+// transaction, however, it is not safe to execute other writer transactions at
+// the same time.
+func (tx *Tx) Check(kvStringer KVStringer) <-chan error {
```

Comment:
This is a breaking change. It changed the signature of method Check. Previously there was no parameter.  FYI. https://github.com/etcd-io/etcd/pull/15193/commits/87c0da80e507514404724dc63caa9597b3893325

The proposals:
1. keep it as it's, and add a breaking change note when we release v1.3.7;
2. revert the change, and add a separate method something like `CheckWithStringer(kvString KVString)`, and use a default `KVString` (e.g. `bolt.HexKVStringer`) when calling `Check()`;
3. allow users to pass a `nil` KVStringer, and we use a default one in that case. 

"2" seems to be the safest solution, because it has no any impact on users.